README.md

```diff
@@ -60,7 +63,29 @@ Then navigate to https://github.com/elaskavaia/bga-sharedcode/pulls, click "New
         ".vscode",
         ".git",
         ".DS_Store",
-        "misc/"
-    ]
+        "misc/",
+        "node_modules",
+        "package-lock.json"
+    ],
+    "syncOption": {
+        "skipCreate": false,
+        "delete": true 
+    }
 }
 ```
+- Sometimes you will have to run a manual push of your files and you can do that by using the command shortcut `Ctrl+Shift+P` and runing `SFTP: Sync Local -> Remote`
+- Install Run on Save extension (https://marketplace.visualstudio.com/items?itemName=pucelle.run-on-save)
+- Configure the Run on Save extention by clickin the Extensions tab in VSCode -> Gear icon on Run on Save -> Extension Settings -> Edit settings.json -> copy this into that file
+```
+"runOnSave.commands": [
+        {
+            "match": ".*\\.scss$",
+            "command": "sass ${file} ${fileDirname}/${fileBasenameNoExtension}.css",
+            "runIn": "backend",
+            "runningStatusMessage": "Compiling ${fileBasename}",
+            "finishStatusMessage": "${fileBasename} compiled"
+        }
+    ]
+```
+- Every time you edit and save in the py generator or the scss file, it will compile into css read to be used.
```

Comment:
This still references the python file.

Reply:
ah yes I think I lost that note to re-review this file. sorry, embaressing.